staging/src/kubevirt.io/api/snapshot/v1alpha1/types.go

```diff
@@ -121,6 +121,20 @@ type VirtualMachineSnapshotStatus struct {
 	// +optional
 	// +listType=set
 	Indications []Indication `json:"indications,omitempty"`
+
+	// +optional
+	SnapshotVolumes *SnapshotVolumesLists `json:"snapshotVolumes,omitempty"`
+}
+
+// SnapshotableSourceTypeVolumesLists includes the list of volumes which were included in the snapshot and volumes which were excluded from the snapshot
```

Comment:
Typo: The comment still refers to the previous name

Reply:
Thanks for noticing!